java-checks-testkit/src/main/java/org/sonar/java/checks/verifier/internal/Expectations.java

```diff
@@ -180,17 +189,77 @@ public String toString() {
     }
   }
 
+  private static class EditComment {
```

Comment:
Maybe this name is not the best after all. What about `QuickFixEditComment` or simply `QuickFixEdit`?